sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/changelog.md

```diff
@@ -2,7 +2,15 @@
 
 ## Current version
 ###  Feature Additions
+
+## Version 4.2.0
+###  Feature Additions
+* Added support for Data Flow:
+    - Added data flow object model and CRUD APIs
+    - Added data flow debug session APIs
+    - Added support for execute data flow activity type
 - Added property queryTimeout to tabular sources
```

Comment:
This line belongs to 4.1.4 -- see #7734 
Please add a title and put it there.

Reply:
plz see above